### PR TITLE
added initials support

### PR DESCRIPTION
**Pull request by acaldas**
https://github.com/ripe-tech/ripe-core/issues/2913

### CODE REVIEW

File: src/js/base/ripe.js [hunk 1]
```diff
@@ -20,6 +20,8 @@ ripe.Ripe.prototype.init = function(brand, model, options) {
     this.variant = this.options.variant || null;
     this.url = this.options.url || "https://sandbox.platforme.com/api/";
     this.parts = this.options.parts || {};
+    this.initials = "";
+    this.engraving = null;
```
Comment (joamag) [inline src/js/base/ripe.js]: let's not use the engraving term in the SDK, just initials and profiles

**Comment by joamag** [comment]
@acaldas let's review this manually tomorrow

**Comment by acaldas** [comment]
@joamag the profile is now set using a profileBuilder function. How should the initials be transformed? Like when they have to be uppercase or have dots.

File: src/python/ripe_demo/static/js/ripe.js [hunk 2]
```diff
@@ -1532,6 +1568,21 @@ ripe.Image.prototype.setFrame = function(frame, options) {
     this.update();
 };
 
+ripe.Image.prototype.setInitials = function(initials, options) {
```
Comment (joamag) [inline src/python/ripe_demo/static/js/ripe.js]: maybe set the engraving together with initials.

**Pull request merged**